package.json

```diff
@@ -26,7 +26,9 @@
     "lint": "eslint --ignore-path ./.eslintignore ./src",
     "lint:fix": "eslint ./src --fix",
     "storybook": "start-storybook -p 9009 -s public",
-    "build-storybook": "build-storybook -s public"
+    "build-storybook": "build-storybook -s public",
+    "audit:resolve": "SHELL=sh resolve-audit && sed -i 's/: \"^/: \"/' package.json",
```

Comment:
If this could use `package-lock.json`, it would be better to change. If it requires actual work to the tool, we can keep it as-is and deal with the narrowest of edge cases as they arise.

Reply:
Ah, nah this just makes sure that no carets get added into the `package.json` file after the resolve-audit step is done, so it doesn't apply to the `package-lock.json` file